src/Facility.CodeGen.Python/template.scriban-txt

```diff
@@ -1,206 +1,265 @@
-{{ func WriteSummary ~}}
-{{ if !$0.Summary.empty? ~}}
 
-{{ $0.Summary }}
+{{ func WriteFieldValue # $0 source depth ~}}{{ if $0.Kind == 9 }}{{ $0.Dto.Name }}.from_data({{ $1 }}){{ else if $0.Kind == 2 ~}}float({{ $1 }}){{ else if $0.Kind == 3 || $0.Kind == 4 ~}}int({{ $1 }}){{ else if $0.Kind == 5 ~}}decimal.Decimal({{ $1 }}){{ else if $0.Kind == 6 ~}}base64.b64decode({{ $1 }}){{ else if $0.Kind == 8 ~}}facility.Error.from_data({{ $1 }}){{ else if $0.Kind == 10 && $0.Enum ~}}{{ $0.Enum.Name }}.get({{ $1 }}){{ else if $0.Kind == 11 ~}}facility.Result.from_data({{ $1 }}){{ else if $0.Kind == 12 && $0.Dto ~}}[{{ WriteFieldValue $0.Dto ("v"+$2) ($2+1) }} for {{ ("v"+$2) }} in {{ $1 }}]{{ else if $0.Kind == 12 && $0.ValueType ~}}[{{ WriteFieldValue $0.ValueType ("v"+$2) ($2+1) }} for {{ ("v"+$2) }} in {{ $1 }}]{{ else ~}}{{ $1 }}{{ end ~}}{{ end # func WriteFieldValue ~}}
```

Comment:
It might be nice to spread this onto multiple lines.

Reply:
I agree but needed to avoid inserting line breaks, while allowing line breaks to be significant elsewhere. Perhaps I can do better.